src/functions/process.go

```diff
@@ -78,10 +80,18 @@ func processKeyValues(keyValues []string) {
 
 	for _, pair := range keyValues {
 		key, value := parseKeyValuePair(pair)
-		if key == "" {
+		if key == constants.EmptyAsString {
 			continue
 		}
 
+		keyAsString := utils.ToAbsoluteString(value)
+		keyIsLangOrPlatform := key == "langs" || key == "platforms"
```

Comment:
```suggestion
		keyLangOrPlatform := key == "langs" || key == "platforms"
```

Reply:
No hace mucho sentido que la variable sea "is", cuando en el uso, pregutamos lo mismo con `if` nuevamente:
```go
if (keyLangOrPlatform) // Es lang o plataforma... Sino queda "is" "is"
```

Reply:
Valid